### ISSUE TITLE
Option for disabling wrap scanning (nowrapscan)?

### ISSUE THREAD

**Issue by catzybluphish** [philc/vimium#3553]
Is there an option for disabling wrap scanning when searching for terms? You can disable this with `set nowrapscan` in your vimrc. If not, I'm requesting this feature.

**Comment by dumblob** [comment]
I'm in need for this as well. I saw @gdh1995 probably implemented it in https://github.com/gdh1995/vimium-c/issues/13 but maybe I'm wrong.

@gdh1995 could you comment on this please?

**Comment by dumblob** [comment]
Just found this: https://github.com/philc/vimium/issues/3172#issuecomment-516343238 . So it seems it's more about a bug in Firefox than about Vimium :cry:.

**Comment by gdh1995** [comment]
Yes Firefox doesn't support the feature of wrap finding in its JavaScript APIs. This can be simulated by a further step: move selection to head/end and search again, just like what I've added into my Vimium C (https://github.com/gdh1995/vimium-c)

**Comment by gdh1995** [comment]
In my Vimium C (https://github.com/gdh1995/vimium-c), `\A` in Find HUD means to turn off `wrapAround`, and `\a` can re-enable it (if not in regexp mode).
